examples/plot_geodesic_poincare_polydisk.py

```diff
@@ -0,0 +1,71 @@
+"""
+Plot a geodesic on the Poincare polydisk,
+with Poincare Disk visualization.
+"""
+
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geomstats.backend as gs
+import geomstats.visualization as visualization
+from geomstats.geometry.poincare_polydisk import PoincarePolydisk
+
+two_disks = PoincarePolydisk(n_disks=1, point_type='extrinsic')
+METRIC = two_disks.metric
+
+
+def plot_geodesic_between_two_points(initial_point,
+                                     end_point,
+                                     n_steps=10,
+                                     ax=None):
+    assert two_disks.belongs(initial_point)
+    assert two_disks.belongs(end_point)
+
+    geodesic = METRIC.geodesic(initial_point=initial_point,
+                               end_point=end_point)
+
+    t = np.linspace(0, 1, n_steps)
+    points = geodesic(t)
+    visualization.plot(points, ax=ax, space='Poincare polydisk (for 2 disks)')
```

Comment:
```space='poincare_polydisk'```

and modify visualization.py, the plot function

Reply:
fig = plt.figure(figsize=(10, 5))

for i_disk in range(n_disks):
    ax_i = fig.add_subplot()